### PR TITLE
Feature/implement snackbar feedback

### PR DESCRIPTION
**Pull request by balint-szabox**
feature/register error messages design and some logic:
- Design to the Error messages, thez weren't super visibile before
- Feedback on successful registration with Snackbar and we navigete to the Login page also
- SnackbarProvider added to the App

feature/Login feedback messages:
- TextFields in Login and Register file got a small design update to match the rest of the app
- Better Alert messages and logic
- Feedback on successful Login with Snackbar

### CODE REVIEW

File: src/components/SnackbarContext.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const SnackbarProvider: React.FC<SnackbarProviderProps> = ({
       {children}
       <Snackbar
         open={open}
-        autoHideDuration={6000}
+        autoHideDuration={3000}
```
Comment (bormilan) [inline src/components/SnackbarContext.tsx]: we should use variables in the code and assign the values in the variables.

File: src/scenes/login/Login.tsx [hunk 2]
```diff
@@ -60,15 +62,29 @@ const LoginComponent = () => {
       }
       setAlertMessage("Login was successful!");
       setAlert(true);
+      setTimeout(() => {
+        setAlert(false);
+        setLoginSuccessfull(false);
+      }, 5000);
```
Comment (bormilan) [inline src/scenes/login/Login.tsx]: why setting timeout ?

**Review by bormilan — CHANGES_REQUESTED** [review]

File: src/components/SnackbarContext.tsx (hunk 1 above)
Comment (balint-szabox) [reply src/components/SnackbarContext.tsx]: I've refactored the code to use variables for better readability and maintainability. Now, all the values are assigned to variables. I've tested the changes to ensure that they work.

File: src/scenes/login/Login.tsx (hunk 2 above)
Comment (balint-szabox) [reply src/scenes/login/Login.tsx]: I removed the specific code. With that change, the timeout has reverted back to the default 3 seconds, which seems to be working well for our use case

**Review by bormilan — APPROVED** [review]

**Pull request merged**